app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -281,16 +282,12 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.findItem(R.id.action_redo).setVisible(_appSettings.isEditorHistoryEnabled());
         menu.findItem(R.id.action_send_debug_log).setVisible(MainActivity.IS_DEBUG_ENABLED && getActivity() instanceof DocumentActivity && !_isPreviewVisible);
 
-        final boolean canUndo = _editTextUndoRedoHelper.getCanUndo();
-        final boolean canRedo = _editTextUndoRedoHelper.getCanRedo();
         final boolean isExperimentalFeaturesEnabled = _appSettings.isExperimentalFeaturesEnabled();
 
         // Undo / Redo / Save (keep visible, but deactivated and tinted grey if not executable)
-        Drawable drawable;
-        drawable = menu.findItem(R.id.action_undo).setEnabled(canUndo).setVisible(!_isPreviewVisible).getIcon();
-        drawable.mutate().setAlpha(canUndo ? 255 : 40);
-        drawable = menu.findItem(R.id.action_redo).setEnabled(canRedo).setVisible(!_isPreviewVisible).getIcon();
-        drawable.mutate().setAlpha(canRedo ? 255 : 40);
+        _undoMenuItem = menu.findItem(R.id.action_undo).setVisible(!_isPreviewVisible);
+        _redoMenuItem = menu.findItem(R.id.action_redo).setVisible(!_isPreviewVisible);
+        _saveMenuItem = menu.findItem(R.id.action_save).setVisible(!_isPreviewVisible);
```

Comment:
So it is hidden now as I see it, but later below there is still the code `setEnabled => AlphaTint`

![image](https://user-images.githubusercontent.com/6735650/147339334-0bbe08bb-2095-4f31-8e06-0bed7095fee1.png)

Can I remove tinting there then?